client/my-sites/plans-features-main/index.jsx

```diff
@@ -197,7 +197,7 @@ class PlansFeaturesMain extends Component {
 				<FAQItem
 					question={ translate( 'Do you sell domains?' ) }
 					answer={ translate(
-						'Yes! The personal, premium, and business plans include a free custom domain. That includes new' +
+						'Yes! The Personal, Premium, and Business plans include a free custom domain. That includes new' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Yes! The personal, premium, and business plans include a free custom domain. That includes new domains purchased through WordPress.com or your own existing domain that you can map to your WordPress.com site. {{a}}Find out more about domains.{{/a}}' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).